src/main.ts

```diff
@@ -993,4 +1124,4 @@ async function downloadTsNpmAsync(cwd: string, version: string, entrypoint: TsEn
     }
 
     return { tsEntrypointPath, resolvedVersion };
-}
+}
```

Comment:
```suggestion
}

```

Reply:
This one shows weird but mainly it's that the newline got removed from the end here which seems unintentional.